src/tools/crossgen2/ILCompiler.ReadyToRun/JitInterface/CorInfoImpl.ReadyToRun.cs

```diff
@@ -1842,5 +1842,7 @@ private bool TypeRequiresMarshaling(TypeDesc type, bool isReturnType)
 
             return false;
         }
+
+        private int SizeOfPInvokeTransitionFrame => 11 * _compilation.NodeFactory.Target.PointerSize;
```

Comment:
I am wondering whether this constant should be in src\inc\readytorun.h to make it clear that it is part of the R2R contract. (Of course, it would not help here - because of the C++ header file cannot be included from here.)